docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
+
+## **Overview**
+This document provides a list of public IPs used by Pipes for outbound requests. These IPs are intended for end-users to configure whitelisting, firewall rules, and integrations with external services.
+
+## **Public IPs**
+#### **1. Load Balancer IPs**
+The following IPs are used by Pipes' load balancers. Please ensure these are whitelisted for access to our services:
+
+| **Load Balancer Name**       | **Static IP Address**     | **Ports**     |
+|------------------------------|---------------------------|---------------|
+| Main Service           | 34.86.218.179            | 80, 443       |
+| Database Proxy                    | 34.145.175.31            | 9193          |
+| Dashboard                    | 34.85.232.246            | 80, 443       |
```

Comment:
As above - `Powerpipe Dashboard` or `Powerpipe Dashboard Proxy`?

Reply:
Updated to `Powerpipe Dashboard Proxy`